core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7706,8 +7706,7 @@ private static BiFunction<String, PropertyKey, PropertyKey> createNestedProperty
     private final PropertyType mType;
     private final Optional<Class<? extends Enum>> mEnumType;
     private final Optional<String> mDelimiter;
-    private BiFunction<String, PropertyKey, PropertyKey> mPropertyCreator =
-        PropertyCreators.DEFAULT_PROPERTY_CREATOR;
```

Comment:
Please also delete the definition of `DEFAULT_PROPERTY_CREATOR`. I don't think it's used anywhere else.

Reply:
Thanks for reminder, removed.